tests/test_decorators_roles.py

```diff
@@ -0,0 +1,125 @@
+from models.decorators_roles import (
+    require_logged_in,
+    require_not_logged_in,
+    require_role
+)
+import pytest
+from models.users import Role, RoleEnum
+from mimesis import Generic
+from flask import session
```

Comment:
It's a minor style issue, but we prefer ordered, grouped imports in that project: python std lib, python libs, flask imports, flask libs and app modules imports.